be/src/vec/olap/olap_data_convertor.cpp

```diff
@@ -786,6 +786,12 @@ Status OlapBlockDataConvertor::OlapColumnDataConvertorArray::convert_to_olap(
     return Status::OK();
 }
 
+void OlapBlockDataConvertor::OlapColumnDataConvertorMap::set_source_column(
```

Comment:
Is necessary to override set_source_column and just call parent's set_source_column as impl?